### ISSUE TITLE
[Library] Own library on Android/Shield causes exception

### ISSUE THREAD

**Issue by maniacgit** [asciidisco/plugin.video.netflix#83]
If I enable the own library in the options and try to open the plugin afterwards I get this exception on Android on Nvidia Shield:

```
12:57:19.702 T:138159989840   ERROR: EXCEPTION Thrown (PythonToCppException) : -->Python callback/script returned the following error<--
                                             - NOTE: IGNORING THIS CAN LEAD TO MEMORY LEAKS!
                                            Error Type: <type 'exceptions.OSError'>
                                            Error Contents: [Errno 30] Read-only file system: 'special:'
                                            Traceback (most recent call last):
                                              File "/storage/emulated/0/Android/data/org.xbmc.kodinerds/files/.kodi/addons/plugin.video.netflix/addon.py", line 23, in <module>
                                                log_fn=kodi_helper.log
                                              File "/storage/emulated/0/Android/data/org.xbmc.kodinerds/files/.kodi/addons/plugin.video.netflix/resources/lib/Library.py", line 57, in __init__
                                                self.series_label: self.tvshow_path
                                              File "/storage/emulated/0/Android/data/org.xbmc.kodinerds/files/.kodi/addons/plugin.video.netflix/resources/lib/Library.py", line 73, in setup_local_netflix_library
                                                os.makedirs(source[label])
                                              File "/opt/dev/kodi/android/kodinerds-master-aarch64/tools/depends/xbmc-depends/aarch64-linux-android-android-21-release/lib/python2.7/os.py", line 150, in makedirs
                                              File "/opt/dev/kodi/android/kodinerds-master-aarch64/tools/depends/xbmc-depends/aarch64-linux-android-android-21-release/lib/python2.7/os.py", line 150, in makedirs
                                              File "/opt/dev/kodi/android/kodinerds-master-aarch64/tools/depends/xbmc-depends/aarch64-linux-android-android-21-release/lib/python2.7/os.py", line 150, in makedirs
                                              File "/opt/dev/kodi/android/kodinerds-master-aarch64/tools/depends/xbmc-depends/aarch64-linux-android-android-21-release/lib/python2.7/os.py", line 150, in makedirs
                                              File "/opt/dev/kodi/android/kodinerds-master-aarch64/tools/depends/xbmc-depends/aarch64-linux-android-android-21-release/lib/python2.7/os.py", line 157, in makedirs
                                            OSError: [Errno 30] Read-only file system: 'special:'
                                            -->End of Python script error report<--
```

**Comment by asciidisco** [comment]
@maniacgit What path did you choose for storing the files?

**Comment by maniacgit** [comment]
I've tried it with the defalut path special://profile/addon_data/plugin.video.netflix

**Comment by asciidisco** [comment]
Should be fixed in version 0.11.24
